controllers/operatorpolicy_controller.go

```diff
@@ -176,3 +248,58 @@ func (r *OperatorPolicyReconciler) shouldEvaluatePolicy(
 
 	return true
 }
+
+// buildSubscription bootstraps the subscription spec defined in the operator policy
+// with the apiversion and kind in preparation for resource creation
+func (r *OperatorPolicyReconciler) buildSubscription(
```

Comment:
I don't think this will ever need anything from the Reconciler object, so this could be changed to just a function, as opposed to a method. The advantage to that is that it is then a bit easier to test in a unit test, because it's obvious that you don't need to instantiate a client or anything.

The CI is currently failing for the PR, because it is just barely missing the test coverage goal. I think adjusting this and `buildOperatorGroup` to be regular functions, and adding a unit test for each, would solve this nicely. Eventually of course we'll want e2e tests for OperatorPolicies.